src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -159,7 +159,7 @@ public RuntimeConfig(string? Schema, DataSource DataSource, RuntimeEntities Enti
         this.DataSource = DataSource;
         this.Runtime = Runtime;
         this.Entities = Entities;
-        _defaultDataSourceName = Guid.NewGuid().ToString();
+        _defaultDataSourceName = "DEFAULT_DATASOURCE_NAME";
```

Comment:
Changed this to a constant value so that we don't have to save the unique ID for reloads. We have correlation ID to track individual requests already, not sure having unique UIDs here grants us any advantage.